### ISSUE TITLE
inference results with noise

### ISSUE THREAD

**Issue by winnechan** [lucidrains/denoising-diffusion-pytorch#45]
Hi, thanks for your excellent project !

I am new to diffusion models and I am now training a vanilla diffusion model over a small dataset (100 images). After being trained for 3000 epoches (around 10k iterations) with initial learning rate 5e-5, the trained model with small training loss produces images with much noises just like 

![bd52d51a-63a5-46f8-94a4-230ba55ca19a](https://user-images.githubusercontent.com/9052195/174985844-545fb828-51ad-4c4e-9e47-ce063f887a35.jpg)

How can I resolve the problem ? Should I train it for longer time or enlarge the learning rate ? Some useful tricks to debug ? 

Appreciate any help !

**Comment by lucidrains** [comment]
@winnechan Hi Winne! So 100 images may not be enough, but eventually I want to add the augmentation techniques from https://arxiv.org/abs/2206.00364, and that should allow DDPMs to work on small datasets just as well

**Comment by HenkPoley** [comment]
Birch-san implemented arXiv:2206.00364 for a stable-diffusion fork:

https://github.com/Birch-san/stable-diffusion/compare/6a74a6aa17026fbb2c7fdfdb803dba591427a027...bbbb1cb81fe4d09cd5d47390e068e05baf5332bb
